pom.xml

```diff
@@ -97,6 +97,30 @@
 				</plugin>
 			</plugins>
 		</pluginManagement>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>2.17</version>
+				<executions>
+					<execution>
+						<id>checkstyle-validation</id>
+						<phase>validate</phase>
+						<configuration>
```

Comment:
If you move this configuration outside of the `<execution>`, you'll be able to run checkstyle with just `mvn checkstyle:check`.